web/turkey/survey/static/survey/js/auditors/clicks_specific.js

```diff
@@ -0,0 +1,29 @@
+var AuditorClicksSpecifics = {
+    clicks_specifics: [],
+    log_click_content: function (e) {
+        e.stopPropagation();
+        this.clicks_specifics.push(e.target);
+    },
+    submit_callable: function () {
+        return {
+            'content':  this.clicks_specifics != []
```

Comment:
This will be easier to handle on the Pythond end if you don't set 'content' to null when the list is empty. That will be converted to the Python singleton None by Django Rest Framework, so when there is an iterator like 
`for click in clicks:
     do stuff`
the iterator will crash on None. So None will become an edge case which requires special handling code. Whereas, if it is left as an empty list, the iterator will simply be skipped and the presence of edge cases avoided.